src/Storage/Masa.Contrib.Storage.ObjectStorage.Aliyun/Options/AliyunOptions.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Contrib.Storage.ObjectStorage.Aliyun.Options;
+
+public class AliyunOptions
+{
+    public string AccessKeyId { get; set; }
+
+    public string AccessKeySecret { get; set; }
+
+    /// <summary>
+    /// sts region id
```

Comment:
Why is `AliyunOptions.RegionId` comment `sts region id`?

Reply:
The RegionIds of different Alibaba Cloud services are not exactly the same. It is more appropriate to use StsRegionId here. However, since it has been set as RegionId earlier, only the parameter annotations are changed here.

Reply:
processed